source/braille.py

```diff
@@ -2087,6 +2101,46 @@ def terminate(self):
 			self.ackTimerHandle = None
 		louisHelper.terminate()
 
+	# The list containing the regions that will be shown in braille when the speak function is called
+	# and the braille mode is set to speech output
+	_showSpeechInBrailleRegions: list[TextRegion] = []
+
+	def _showSpeechInBraille(self, speechSequence: "SpeechSequence"):
+		if config.conf["braille"]["mode"] == BrailleMode.FOLLOW_CURSORS.value:
+			return
+		_showSpeechInBrailleRegions = self._showSpeechInBrailleRegions
+		regionsText = "".join([i.rawText for i in _showSpeechInBrailleRegions])
+		if len(regionsText) > 100000:
+			return
+		text = " ".join([x for x in speechSequence if isinstance(x, str)])
+		currentRegions = False
+		if _showSpeechInBrailleRegions:
+			text = f" {text}"
+			currentRegions = True
+
+		region = TextRegion(text)
+		region.update()
+		_showSpeechInBrailleRegions.append(region)
+		self.mainBuffer.regions = _showSpeechInBrailleRegions.copy()
+		if not currentRegions:
+			handler.mainBuffer.focus(_showSpeechInBrailleRegions[0])
+		handler.mainBuffer.update()
+		handler.update()
+
+	_suppressClearBrailleRegions: bool = False
+
+	@contextlib.contextmanager
+	def suppressClearBrailleRegions(self, script: inputCore.InputGestureScriptT):

```

Comment:
I don't believe `script` is a tuple here? I don't believe this is the correct type hint, please see comment 

https://github.com/nvaccess/nvda/pull/15956#discussion_r1480646802

Reply:
please fix up this concern